packages/babel-plugin-fbtee/src/bin/manifestUtils.tsx

```diff
@@ -23,17 +26,31 @@ export async function generateManifest(
   const enumManifest: {
     [enumModuleName: string]: EnumModule;
   } = {};
+
   for (const src of srcPaths) {
     const enumFiles: Array<string> = globSync(
       resolve(cwd, src) + '/**/*' + ENUM_FILE + FILE_EXT,
       {
         nodir: true,
       },
     );
+
     for (const filepath of enumFiles) {
       const name = parse(filepath).name;
-      const obj = (await import(resolve(filepath))).default;
-      const enumValue: EnumModule = obj.__esModule ? obj.default : obj;
+      const tsContent = fs.readFileSync(filepath, 'utf8');
+      const result = transformSync(tsContent, {
+        ast: false,
+        filename: filepath,
+        presets: [presetTypeScript, presetReact],
+        sourceType: 'module',
+      });
+
+      invariant(result?.code, 'Unable to transform `%s`', filepath);
+
+      const jsDataUrl = `data:text/javascript;base64,${Buffer.from(result.code).toString('base64')}`;
```

Comment:
TIL 🙂
Nifty way to import dynamically generated JS code.

Reply:
@alexandernanberg why did you make this change? Shouldn't the manifest always be JSON? If not, then we can apply the same solution you used for enums. You can also check out this example for common strings which can be JSON or JS/TS: https://github.com/nkzw-tech/fbtee/blob/main/packages/babel-plugin-fbtee/src/bin/collect.tsx#L246-L253 – might even be worth turning into a helper function.

Reply:
This was before we made the change with node flags. I've now rebased on main so it's gone!